__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -13,7 +13,7 @@ const {
 import { NESTED_COLUMNS_3_LEVELS } from './test-editor-data';
 
 const ANDROID_COLUMN_APPENDER_BUTTON_XPATH =
-	'//android.widget.Button[@content-desc="Column Block. Row 1"]/android.view.ViewGroup[2]/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.widget.Button';
```

Comment:
@SiobhyB it appears I originally defined this problematic XPath. I recall lacking confidence in its definition at the time. For my own education, what was your approach for identifying this simpler, more stable XPath? Thanks!

Reply:
@dcalhoun, I don't necessarily think the previous XPath was problematic, rather I think something must have changed in the view hierarchy, as it had begun to fail consistently. To get the newer XPath, I followed a process that I believe you may have also followed i.e. I selected the component in [appium-inspector](https://github.com/appium/appium-inspector) and then copied/pasted the current XPath value.

(Note, you'll see in [my first commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5986/commits/7f5e9c1a69e7aa56b147924185b7da6973f2c6cf) that I'd initially copied/pasted the _entire_ XPath. I referenced simplifying that XPath in [a later commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5986/commits/c9aa8933441b14d871a39992fb3e4dba68fd1835) when I realised it wasn't needed in its entirety. The references to stability and simplification in that later commit's message are related to my mistake within this PR, not the test's previous XPath.)

Let me know if I can further expand on anything!

Reply:
Ah, OK. Yes, I sourced my XPath from Appium Inspector as well. Thanks for elaborating!